diff --git a/docshell/base/nsAboutRedirector.cpp b/docshell/base/nsAboutRedirector.cpp
index a13b07368c8ea1b7510d55ac9e3dcae50f226158..0aad5269eb207ccd0974688ac27062c195230402 100644
--- a/docshell/base/nsAboutRedirector.cpp
+++ b/docshell/base/nsAboutRedirector.cpp
@@ -117,6 +117,15 @@ static const RedirEntry kRedirMap[] = {
     {"crashes", "chrome://global/content/crashes.html",
      nsIAboutModule::IS_SECURE_CHROME_UI},
 #endif
+    {"limb-home", "chrome://browser/content/limb/home/limb-home.html",
+     nsIAboutModule::IS_SECURE_CHROME_UI |
+         nsIAboutModule::ALLOW_SCRIPT},
+    {"limb-search", "chrome://browser/content/limb/search/limb-search.html",
+     nsIAboutModule::IS_SECURE_CHROME_UI |
+         nsIAboutModule::ALLOW_SCRIPT},
+    {"limb-settings", "chrome://browser/content/limb/settings/limb-settings.html",
+     nsIAboutModule::IS_SECURE_CHROME_UI |
+         nsIAboutModule::ALLOW_SCRIPT},
     {"credits", "https://zen-browser.app/about/",
      nsIAboutModule::URI_SAFE_FOR_UNTRUSTED_CONTENT |
          nsIAboutModule::URI_MUST_LOAD_IN_CHILD},
