diff --git a/browser/components/BrowserGlue.sys.mjs b/browser/components/BrowserGlue.sys.mjs
index cb4f544e3c..d014592460 100644
--- a/browser/components/BrowserGlue.sys.mjs
+++ b/browser/components/BrowserGlue.sys.mjs
@@ -8,7 +8,6 @@ import { XPCOMUtils } from "resource://gre/modules/XPCOMUtils.sys.mjs";
 const lazy = {};
 
 ChromeUtils.defineESModuleGetters(lazy, {
-  gZenUIMigration: "resource:///modules/ZenUIMigration.sys.mjs",
   AboutHomeStartupCache: "resource:///modules/AboutHomeStartupCache.sys.mjs",
   AWToolbarButton: "resource:///modules/aboutwelcome/AWToolbarUtils.sys.mjs",
   ASRouter: "resource:///modules/asrouter/ASRouter.sys.mjs",
@@ -1622,8 +1621,6 @@ BrowserGlue.prototype = {
     } else if (profileDataVersion < APP_DATA_VERSION) {
       lazy.ProfileDataUpgrader.upgrade(profileDataVersion, APP_DATA_VERSION);
     }
-
-    lazy.gZenUIMigration.init(this._isNewProfile);
   },
 
   async _showUpgradeDialog() {
