diff --git a/browser/components/sessionstore/SessionSaver.sys.mjs b/browser/components/sessionstore/SessionSaver.sys.mjs
index 6906fd9be7..4af2793f80 100644
--- a/browser/components/sessionstore/SessionSaver.sys.mjs
+++ b/browser/components/sessionstore/SessionSaver.sys.mjs
@@ -20,7 +20,6 @@ ChromeUtils.defineESModuleGetters(lazy, {
   SessionFile: "resource:///modules/sessionstore/SessionFile.sys.mjs",
   SessionStore: "resource:///modules/sessionstore/SessionStore.sys.mjs",
   sessionStoreLogger: "resource:///modules/sessionstore/SessionLogger.sys.mjs",
-  ZenSessionStore: "resource:///modules/zen/ZenSessionManager.sys.mjs",
 });
 
 /*
@@ -306,7 +305,7 @@ var SessionSaverInternal = {
     this._maybeClearCookiesAndStorage(state);
 
     Glean.sessionRestore.collectData.stopAndAccumulate(timerId);
-    lazy.ZenSessionStore.saveState(state);
+// [Zen removed] lazy.ZenSessionStore.saveState(state);
     return this._writeState(state);
   },
 
