diff --git a/browser/components/sessionstore/SessionStartup.sys.mjs b/browser/components/sessionstore/SessionStartup.sys.mjs
index 70d7b5c7d6..9189dd9559 100644
--- a/browser/components/sessionstore/SessionStartup.sys.mjs
+++ b/browser/components/sessionstore/SessionStartup.sys.mjs
@@ -40,7 +40,6 @@ ChromeUtils.defineESModuleGetters(lazy, {
   StartupPerformance:
     "resource:///modules/sessionstore/StartupPerformance.sys.mjs",
   sessionStoreLogger: "resource:///modules/sessionstore/SessionLogger.sys.mjs",
-  ZenSessionStore: "resource:///modules/zen/ZenSessionManager.sys.mjs",
 });
 
 const STATE_RUNNING_STR = "running";
@@ -180,7 +179,7 @@ export var SessionStartup = {
       this._initialState = parsed;
     }
 
-    this._initialState = lazy.ZenSessionStore.onFileRead(this._initialState);
+// [Zen removed] this._initialState = lazy.ZenSessionStore.onFileRead(this._initialState);
 
     if (this._initialState == null) {
       // No valid session found.
@@ -279,7 +278,7 @@ export var SessionStartup = {
         `Previous shutdown ok? ${this._previousSessionCrashed}, reason: ${previousSessionCrashedReason}`
       );
 
-      lazy.ZenSessionStore.onCrashCheckpoints(this._initialState);
+// [Zen removed] lazy.ZenSessionStore.onCrashCheckpoints(this._initialState);
 
       Services.obs.addObserver(this, "sessionstore-windows-restored", true);
 
