diff --git a/browser/base/content/browser.js b/browser/base/content/browser.js
index 37db181d7e..d44ed8a90f 100644
--- a/browser/base/content/browser.js
+++ b/browser/base/content/browser.js
@@ -33,7 +33,6 @@ ChromeUtils.defineESModuleGetters(this, {
     "resource://gre/modules/ContextualIdentityService.sys.mjs",
   CustomizableUI:
     "moz-src:///browser/components/customizableui/CustomizableUI.sys.mjs",
-  ZenCustomizableUI: "resource:///modules/ZenCustomizableUI.sys.mjs",
   DevToolsSocketStatus:
     "resource://devtools/shared/security/DevToolsSocketStatus.sys.mjs",
   DownloadUtils: "resource://gre/modules/DownloadUtils.sys.mjs",
