diff --git a/browser/components/customizableui/CustomizableUI.sys.mjs b/browser/components/customizableui/CustomizableUI.sys.mjs
index 400ad196c8..80f0eecbb7 100644
--- a/browser/components/customizableui/CustomizableUI.sys.mjs
+++ b/browser/components/customizableui/CustomizableUI.sys.mjs
@@ -14,7 +14,6 @@ ChromeUtils.defineESModuleGetters(lazy, {
   BrowserUsageTelemetry: "resource:///modules/BrowserUsageTelemetry.sys.mjs",
   CustomizableWidgets:
     "moz-src:///browser/components/customizableui/CustomizableWidgets.sys.mjs",
-  ZenCustomizableUI: "resource:///modules/ZenCustomizableUI.sys.mjs",
   HomePage: "resource:///modules/HomePage.sys.mjs",
   PanelMultiView:
     "moz-src:///browser/components/customizableui/PanelMultiView.sys.mjs",
@@ -367,7 +366,6 @@ var CustomizableUIInternal = {
       lazy.ippEnabled ? "ipprotection-button" : null,
       lazy.resetPBMToolbarButtonEnabled ? "reset-pbm-toolbar-button" : null,
     ].filter(name => name);
-    lazy.ZenCustomizableUI.startup(this);
     this.registerArea(
       CustomizableUI.AREA_NAVBAR,
       {
@@ -1944,16 +1942,14 @@ var CustomizableUIInternal = {
         elem.setAttribute("skipintoolbarset", "true");
       }
     }
-    lazy.ZenCustomizableUI.init(aWindow);
     // Now register all the toolbars
     for (let area of CustomizableUI.areas) {
       let type = CustomizableUI.getAreaType(area);
-      if (type == CustomizableUI.TYPE_TOOLBAR && !area.startsWith("zen-")) {
+      if (type == CustomizableUI.TYPE_TOOLBAR) {
         let node = document.getElementById(area);
         this.registerToolbarNode(node);
       }
     }
-    lazy.ZenCustomizableUI.registerToolbarNodes(aWindow);
     // Handle initial state of vertical tabs.
     if (isVerticalTabs) {
       // Show the vertical tabs toolbar
